Tools/LyTestTools/ly_test_tools/_internal/pytest_plugin/test_tools_fixtures.py

```diff
@@ -86,12 +86,17 @@ def _get_output_path(config):
         logger.debug(f'Custom output_path set to: {str(custom_output_path)}')
         output_path = custom_output_path
     else:
-        # from pytest_runner
-        output_path = os.path.join(os.getcwd(),
+        custom_build_directory = config.getoption('--build-directory', '')
+        if custom_build_directory: # default into known build folder
+            default_output_path = custom_build_directory
```

Comment:
Should this perhaps be defaulting into <custom_build_directory>\build\Testing\ ?

Reply:
It's specifically in \build\Testing\LyTestTools\CTestModuleName ...the latter of which isn't going to be available in this context. I'll try to make it more obvious when the user has directly invoked pytest.

Reply:
I'm so sorry I did a typo, there is no "build" in the path. Should be :<custom_build_directory>/Testing/...